SSZipArchive/minizip/mz_compat.h

```diff
@@ -124,11 +124,7 @@ ZEXPORT void fill_memory_filefunc(zlib_filefunc_def *pzlib_filefunc_def);
 
 /***************************************************************************/
 
-
-// SSZipArchive 2.x+ uses dos_date
-#define MZ_COMPAT_VERSION 120
```

Comment:
This removal is on purpose: since minizip-ng 4 breaks the format of the structure, we may as well directly adopt the naming matching https://github.com/madler/zlib/blob/develop/contrib/minizip/unzip.h